sklearn/calibration.py

```diff
@@ -253,6 +257,11 @@ class _CalibratedClassifier(object):
         corresponds to Platt's method or 'isotonic' which is a
         non-parametric approach based on isotonic regression.
 
+    classes : array-like, shape (n_classes,)
```

Comment:
Unimportant, given that this is a private class, but conventionally we'd add ", optional" to this line